products/oracle-jdk.md

```diff
@@ -26,8 +26,8 @@ releases:
     releaseDate: 2023-09-19
     eol: 2028-09-30
     extendedSupport: 2031-09-30
-    latest: "21"
-    latestReleaseDate: 2023-09-19
+    latest: "21.0.1"
+    latestReleaseDate: 2023-10-17
     link: https://www.oracle.com/java/technologies/javase/21-relnote-issues.html
```

Comment:
As this is the first patch version please remove:
```suggestion
```
This will generate the correct link to 21.0.1 release notes.